### PR TITLE
[MM-60818]  Replace FormattedMarkdownMessage in 'webapp/channels/src/components/delete_category_modal/delete_category_modal.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28508
Jira https://mattermost.atlassian.net/browse/MM-60818
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/delete_category_modal/delete_category_modal.tsx [hunk 1]
```diff
@@ -48,9 +46,9 @@ export default function DeleteCategoryModal(props: Props) {
             confirmButtonClassName={'delete'}
         >
             <span className='delete-category__helpText'>
-                <FormattedMarkdownMessage
+                <FormattedMessage
                     id='delete_category_modal.helpText'
-                    defaultMessage="Channels in **{category_name}** will move back to the Channels and Direct messages categories. You're not removed from any channels."
+                    defaultMessage="Channels in <b>{category_name}</b> will move back to the Channels and Direct messages categories. You're not removed from any channels."
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/delete_category_modal/delete_category_modal.tsx:51]: @Aryakoste can you refer the example PR https://github.com/mattermost/mattermost/pull/28438 
We need to create pass the b variable as XML template

File: webapp/channels/src/i18n/en.json
Comment (M-ZubairAhmed) [inline webapp/channels/src/i18n/en.json:1]: I think there was a problem here, there are too many unrelated deletions here

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/i18n/en.json
Comment (M-ZubairAhmed) [reply webapp/channels/src/i18n/en.json:1]: are you sure you are on latest master? and your local is upto date

Comment (M-ZubairAhmed) [reply webapp/channels/src/i18n/en.json:1]: may be you can try to `npm run clean` and do a refesh of `npm install`

Comment (M-ZubairAhmed) [reply webapp/channels/src/i18n/en.json:1]: also please revert these changes

Comment (Aryakoste) [reply webapp/channels/src/i18n/en.json:1]: I have reverted

Comment (Aryakoste) [reply webapp/channels/src/i18n/en.json:1]: Or i should revert to the old commit i think. will do it. I just reverted the file

Comment (M-ZubairAhmed) [reply webapp/channels/src/i18n/en.json:1]: could you replace the default message with the new one in en.json?

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by larkox — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]
image

**Pull request merged**